geom/src/duration.rs

```diff
@@ -222,6 +222,24 @@ impl Duration {
         }
         s
     }
+
+    /// Returns the largest of the two inputs.
+    pub fn max(self, other: Duration) -> Duration {
```

Comment:
Alternatively, could [`impl std::Ord`](https://doc.rust-lang.org/std/cmp/trait.Ord.html#method.max)

Reply:
Ahh, I've been doing it wrong. `Distance` is the same -- it implements `Ord` and then defines its own min and max. I'll clean this up separately. Thanks!